packages/utils/src/analytics.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This will likely go away once `@edx/frontend-platform` is added back as a peer dependency.